web/src/components/drawer/record/record-panel.tsx

```diff
@@ -342,6 +343,8 @@ export const RecordPanel: React.FC<RecordDrawerProps> = ({
     )
   );
 
+  const isPartialFlow = !record.fields.Bytes && !record.fields.Packets;
```

Comment:
What happens with fully dropped flows ? :thinking: 
I feel we will have DropBytes / Pkts and you will display that message then.

Reply:
Hmm good point, I'll have to check, but I wonder if a fully-dropped flow isn't anyway also a partial flow? ie. is it captured from the TC hook at all?

Reply:
Confirming the fully-dropped flow are (by nature) partial flows, as they are never seen from the TC hook:
![partial-flow](https://github.com/user-attachments/assets/a0a1a6cf-57d7-4bd9-863b-e570d7c22fe7)